CHANGELOG.md

```diff
@@ -109,8 +114,8 @@
 - Force usage of `receiverhelper.NewFactory` to implement `component.ReceiverFactory` (#4338)
 - Force usage of `extensionhelper.NewFactory` to implement `component.ExtensionFactory` (#4338)
 - Move `service/parserprovider` package to `config/configmapprovider` (#4206)
-   - Rename `MapProvider` interface to `Provider`
-   - Remove `MapProvider` from helper names
+  - Rename `MapProvider` interface to `Provider`
+  - Remove `MapProvider` from helper names
```

Comment:
Great improvements (probably automatically done by the IDE), please consider to split "refactoring/cleanup" from actual changes PRs :)

Reply:
Yup, good call.